hydra/resource_hydra_jobset_test.go

```diff
@@ -761,3 +768,50 @@ resource "hydra_jobset" "test" {
   keep_evaluations    = 3
 }`, project, os.Getenv("HYDRA_USERNAME"), jobset)
 }
+
+func testAccHydraJobsetConfigInputEmail(project string, jobset string) string {
+	return fmt.Sprintf(`
+resource "hydra_project" "test" {
+  name         = "%s"
+  display_name = "Ofborg"
+  description  = "ofborg automation"
+  homepage     = "https://github.com/nixos/ofborg"
+  owner        = "%s"
+  enabled = true
+  visible = true
+}
+
+resource "hydra_jobset" "test" {
+  project     = hydra_project.test.name
+  state       = "enabled"
+  visible     = true
+  name        = "%s"
+  type        = "legacy"
+  description = ""
+
+  nix_expression {
+    file  = "release.nix"
+    input = "ofborg"
+  }
+
+  check_interval    = 0
+  scheduling_shares = 3000
+
+  email_notifications = false
+  keep_evaluations    = 3
+
+  input {
+    name              = "nixpkgs"
+    type              = "git"
+    value             = "https://github.com/NixOS/nixpkgs.git nixpkgs-unstable"
+    notify_committers = true
+  }
+
+  input {
+    name              = "ofborg"
+    type              = "git"
+    value             = "https://github.com/nixos/ofborg.git released"
+    notify_committers = false
+  }
```

Comment:
Maybe try setting this field to true on a input type which has no contributors, like a bool or nix expression type, instead of one where we might actually send real people email?